LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIMission_LWCustomMission.uc

```diff
@@ -501,15 +537,17 @@ simulated function BuildGuerrillaOpsOptionsPanel()
 	LibraryPanel.MC.QueueString(""); //GetGOpsMissionLocString(1));
 	LibraryPanel.MC.QueueString(""); //GetGOpsMissionLocString(2));
 	LibraryPanel.MC.QueueString(class'UIUtilities_Text'.default.m_strGenericConfirm);
-	LibraryPanel.MC.QueueString(bCanBackOut ? m_strIgnore : ""); // defined in UIX2SimpleScreen
+	LibraryPanel.MC.QueueString(CanBackOut() ? m_strIgnore : ""); // defined in UIX2SimpleScreen
 	LibraryPanel.MC.EndOp();
 
 	// ----------------------------------------------------------------------
 
+	// WOTC TODO Other mission types may be missing these
 	BuildConfirmPanel();
```

Comment:
What do you think of removing this line? It doesn't do anything unless `LibraryPanel` is `none`, which should never be the case.

Reply:
It is apparently useless, yeah. The only reason I kept it was for symmetry with the original version. That's probably a lot less of an issue now unless they make another xpack or more DLC, but when that happens it's helpful to see exactly what changed from the base game versions and why. I can also just comment it out, with a comment explaining why.

Reply:
Sounds good to me.